collectors/apps.plugin/README.md

```diff
@@ -64,6 +64,27 @@ Each of these sections provides the same number of charts:
     -   Major Page Faults (i.e. swap activity)
 -   Network
     -   Sockets Open
+    
+In addition, if the [eBPF collector](/collectors/ebpf.plugin/README.md) is running, 
+your dashboard will also show the following charts using low-level Linux metrics:
+    
+-   eBPF syscall    
+    -   Number of calls to open files.
+    -   Number of files closed.
+    -   Number of calls to delete files.
+    -   Number of calls to `vfs_write`.
+    -   Number of calls to `vfs_read`.
+    -   Number of bytes written trough `vfs_write`
+    -   Number of bytes read trough `vfs_read`
+    -   Number of process created trough `do_fork`
+    -   Number of threads created trough `do_fork` or `__x86_64_sys_clone`, depending on your system's kernel version.
+    -   Number of times that a process called `do_exit`. 
+    -   Number of calls to open files that returned errors.
+    -   Number of calls to close files that returned errors.
+    -   Number of calls to read a file that returned errors.
+    -   Number of calls to read a file that returned errors.
+-   eBPF net
+    -   Number of bytes transmited per seconds.    
```

Comment:
You should move the list of charts to the eBPF plugin readme. apps plugin doesn't provide these charts. A menu item is not a plugin - it's just a means for visualization.

Reply:
for me it looks like ebpf promoting 😄 

https://github.com/netdata/netdata/pull/9199#discussion_r432177641

kind of acceptable, but in general i agree with you @vlvkobal

Reply:
These charts will be shown in `Apps` submenu, this is the motive we gave a short description here instead the eBPF documentation.